app/assets/stylesheets/modern-business.scss

```diff
@@ -106,7 +106,7 @@ header.carousel .fill#slider-3 {
   font-family: 'Quattrocento', serif;
   font-size: 50px;
   text-align: center;
-  color: #001B4D;
+  color: #A30505;
```

Comment:
Color literals like `#A30505` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A30505` should be written as `#a30505`